containers/kubernetes/reference-content/version-support-policy.mdx

```diff
@@ -42,64 +42,61 @@ Before opening a support ticket related to cluster behavior or a Kubernetes comp
-| 1.26               | December 2022    | February 2024        | February 6, 2023             | February 2025    | March 15, 2025    |
-| 1.25               | August 2022      | October 2023         | February 6, 2023             | February 2025    | March 15, 2025    |
+| 1.29               | December 2023    | February 2025        | February 7, 2024             | February 2025    | April 7, 2025*    |
+| 1.28               | August 2023      | October 2024         | August 2023                  | February 2025*   | April 7, 2025*    |
+| 1.27               | April 2023       | June 2024            | April 2023                   | February 2025*   | April 7, 2025*    |
+| 1.26               | December 2022    | February 2024        | February 6, 2023             | February 2025    | March 6, 2025     |
+| 1.25               | August 2022      | October 2023         | February 6, 2023             | February 2025    | March 6, 2025     |
 | 1.24               | April 2022       | July 2023            | April 2022                   | April 2024       | May 15, 2024      |
 | 1.23               | December 8, 2021 | February 2023        | December 9, 2021             | December 2023    | February 15, 2024 |
 
+<Message type="important">
+* New support policy: from version 1.29 release, Scaleway's support window has shifted to 14 months.
+We're discontinuing extended support for outdated upstream Kubernetes releases to ensure you can always benefit from the latest features, bug fixes, enhanced support and maximum security.
+Consequently, versions 1.27 & 1.28 will have corresponding end of support timelines.
+</Message>
+
 ## Version information
 For a complete changelog update, refer to the official [Kubernetes changelog](https://kubernetes.io/releases/).
 
 The following documentation only lists the main feature changes for each version and provides a link to the version release announcement on Kubernetes' official blog.
 
+### 1.29
+* Official [release announcement](https://kubernetes.io/blog/2023/12/18/kubernetes-1-29-feature-loadbalancer-ip-mode-alpha/)
+* Kubernetes 1.29 includes a total of 11 enhancements promoted to stable, 19 to beta, and 19 to alpha.
+
 ### 1.28
 * Official [release announcement](https://kubernetes.io/blog/2023/08/15/kubernetes-v1-28-release/)
-
-**Main changes:**
 * 45 enhancements: 19 are entering alpha, 14 have graduated to beta, and 12 have graduated to stable.
 
 ### 1.27
 * Official [release announcement](https://kubernetes.io/blog/2023/04/11/kubernetes-v1-27-release/)
-
-**Main changes:**
 * This release consists of 60 enhancements. 18 of those enhancements are entering alpha, 29 are graduating to beta, and 13 are graduating to stable.
 
 ### 1.26
 * Official [release announcement](https://kubernetes.io/blog/2022/12/09/kubernetes-v1-26-release/)
-
-**Main changes:**
-* Eleven features graduated to GA
-* Twelve features deprecated or removed
+* 11 features graduated to GA, 12 features deprecated or removed
```

Comment:
What do you mean by "graduated" ? If you mean "moved to GA", maybe you could use "delayed" or "available during GA" something like that?

Reply:
Graduated is the Kubernetes term for a feature that moves from alpha/beta to prod https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/#feature-gates-for-graduated-or-deprecated-features

Reply:
alpha => beta => GA, each step in between is a graduation!
I've simply switched from a numeral form, the meaning stays the same